Library/Homebrew/compilers.rb

```diff
@@ -1,7 +1,7 @@
 # @private
 module CompilerConstants
-  GNU_GCC_VERSIONS = %w[4.3 4.4 4.5 4.6 4.7 4.8 4.9 5]
-  GNU_GCC_REGEXP = /^gcc-(4\.[3-9]|5)$/
+  GNU_GCC_VERSIONS = %w[4.3 4.4 4.5 4.6 4.7 4.8 4.9 5 6]
```

Comment:
@UniqMartin Is it possible to add a range here too? I'm not familiar enough with ruby syntax to answer that question